src/tests/suite/SConscript

```diff
@@ -5,9 +5,9 @@ def scons():
     """Execute build"""
     Import('denv', 'prereqs')
 
-    libraries = ['daos_common', 'daos', 'daos_tests', 'gurt', 'cart']
+    libraries = ['daos_common', 'daos', 'dfs', 'daos_tests', 'gurt', 'cart']
```

Comment:
i was not able to build this. i did when i added:
denv.AppendUnique(LIBPATH=["$PREFIX/lib/"])
before this.

Reply:
OK, I'll add it, and reduce the test data size.